projects/admin-core/src/lib/catalog/layer-settings-form/layer-settings-form.component.html

```diff
@@ -23,37 +23,85 @@
                                       [layerName]="layerName"
                                       (featureTypeSelected)="updateFeatureTypeSelection($event)"></tm-admin-feature-type-selector>
     </div>
+    <mat-expansion-panel *ngIf="isXYZ">
+      <mat-expansion-panel-header>
+        <mat-panel-title i18n style="flex-grow: 1">XYZ tile grid options</mat-panel-title>
+      </mat-expansion-panel-header>
+<!--
+      TODO: for XYZ layers we don't load capabilities so the projection should be entered by the user. But the admin does not restrict
```

Comment:
https://b3partners.atlassian.net/browse/HTM-872?focusedCommentId=45465 states we will only support webmercator for XYZ